include/qpdf/qpdf-c.h

```diff
@@ -587,6 +587,21 @@ extern "C" {
 
     typedef unsigned int qpdf_oh;
 
+    typedef int QPDF_OT_CODE;
```

Comment:
I won't accept this -- it duplicates information. To implement this, I would want to move object_type_e from QPDFObject.hh to Constants.h and just make QPDFObject::object_type_e an alias to it. I can just do that in my current work and then you can redo this PR to use that.

Reply:
My solution seems like it is also going to duplicate information, but I'm going to use it anyway, so please stay tuned.

Reply:
I didn't like my solution either, but couldn't think of a better one.

Do we actually need to publish the constants in qpdf-c.h? We could have a simple comment that refers QPDFObject.h

I for one won't use the constants but define my own enum. I suspect most other users will do the same.

Reply:
I think it's important to have the enumerated type exposed to the C API if we are going to return values from it. Anyway, I already pushed the change to main.